<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>421</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>캘린더 연동 계정 (Microsoft, outlook)</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonAuth">
        <property name="text">
         <string>인증하기</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>사용자 정보</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>이름</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineName"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>생일</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineBirthday"/>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>직업</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineJob"/>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>분야</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineField"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>158</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QCheckBox" name="check_Organize">
      <property name="text">
       <string>파일정리 기능</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="check_Calendar">
      <property name="text">
       <string>파일 마감 기능</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="check_SendData">
      <property name="text">
       <string>데이터 제공하기</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="check_OBO">
      <property name="text">
       <string>오래된 파일 관리</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="check_Version">
      <property name="text">
       <string>버전 기능</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="check_Package">
      <property name="text">
       <string>패키지 기능</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="ButtonOkay">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>440</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonCancel">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>440</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>취소</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
